src/components/alerting.tsx

```diff
@@ -1618,6 +1618,26 @@ const AlertTableRow_: React.FC<AlertTableRowProps> = ({ history, obj }) => {
     );
   }
 
+  const getExtensionDropdownItems = (actions: Action[]) => {
+    const extensionDropdownItems = actions.map((action) => {
+      if (isActionWithHref(action)) {
+        return (
+          <DropdownItem key={action.id} href={action.cta.href}>
+            {action.label}
+          </DropdownItem>
+        );
+      } else {
+        const onClick = action.cta as () => void;
```

Comment:
Do we need this cast?, at this point `action.cta` can only be a function

Reply:
It was giving me an error if I didn't cast it as a function...

Reply:
Updated as discussed (off thread) with a type guard to check if action is a function. 
```
const isActionWithCallback = (action: Action): action is ActionWithCallBack =>
  typeof action.cta === 'function';
```